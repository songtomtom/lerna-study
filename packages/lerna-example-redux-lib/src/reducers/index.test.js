import rootReducer from './index'

const initialState = rootReducer({}, {})

test('generates correct initial state', () => {
  expect(initialState).toMatchSnapshot();
})
