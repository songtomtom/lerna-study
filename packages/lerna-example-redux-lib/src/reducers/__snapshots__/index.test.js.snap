// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generates correct initial state 1`] = `
Object {
  "facts": Object {
    "isFetching": false,
    "list": Array [],
  },
}
`;
